packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -486,7 +489,7 @@ const CFDPasswordForm = observer(
                                     />
                                 </Text>
                             )}
-                            {account_type.category === CATEGORY.REAL && (
+                            {account_type.category === CATEGORY.REAL && platform === CFD_PLATFORMS.MT5 && (
```

Comment:
```suggestion
                            {need_tnc && (
```

seems like the condition is the same, can just use this
@hasan-deriv svg need tnc?

Reply:
Yeah, can use this...

Reply:
the `need_tnc` has a check for jurisdiction_selected_shortcode, i think it's better to remain this condition 🤔 and we do have info message withour tnc checkbox here: 

![Screenshot 2024-12-16 at 4 05 41 PM](https://github.com/user-attachments/assets/d219451e-90fa-452b-8941-f55d3390a35d)

Reply:
yeah correct @suisin-deriv , we need the info box, just the checkbox depends on svg, ignore my comment then